### PR TITLE
Improve colored strings by using ** to denote bold style

### PR DESCRIPTION
**Pull request by 3lvis**
simulator screen shot 3 jan 2017 14 30 09

### CODE REVIEW

File: Tests/MessageTests.swift [hunk 1]
```diff
@@ -11,4 +12,34 @@ class MessageTests: XCTestCase {
         XCTAssertEqual(Message.Kind.shorterTomorrowMoreThanAMinute, Message.Kind(sunPhase: .night, yesterdayDaylightLength: 0, todayDaylightLength: 61, tomorrowDaylightLength: 0))
         XCTAssertEqual(Message.Kind.shorterTomorrowLessThanAMinute, Message.Kind(sunPhase: .night, yesterdayDaylightLength: 0, todayDaylightLength: 60, tomorrowDaylightLength: 0))
     }
+
+    func testNoAttributesColoredMessage() {
+        let content = "Hello mom."
```
Comment (marijnschilling) [inline Tests/MessageTests.swift]: 😂

File: Tests/MessageTests.swift [hunk 2]
```diff
@@ -11,4 +12,34 @@ class MessageTests: XCTestCase {
...
+
+        let message = Message(format: "Hello **mom**.")
+        XCTAssertEqual(message.content, content)
+        XCTAssertEqual(message.coloredPart, coloredPart)
+
+        let range = (content as NSString).range(of: coloredPart)
+        let attributedString = NSMutableAttributedString(string: content)
+        attributedString.addAttribute(NSForegroundColorAttributeName, value: UIColor.red, range: range)
+
+        XCTAssertEqual(message.attributedString(withTextColor: UIColor.red), attributedString)
```
Comment (marijnschilling) [inline Tests/MessageTests.swift]: What does this test do exactly?

Comment (3lvis) [inline Tests/MessageTests.swift]: Checks that the generated attributed strings is the same as the one we expect.

Comment (marijnschilling) [inline Tests/MessageTests.swift]: But isn't that testing the iOS functions `attributedString` and `addAttribute`?

Comment (3lvis) [inline Tests/MessageTests.swift]: Yes, but if the method `attributedString(withTextColor` is not implemented correctly, this would fail.

Comment (marijnschilling) [inline Tests/MessageTests.swift]: Aha I see, it's one of our own methods ✌️ Just wondering then if we should test this method in a separate test. One test for the .colouredPart to be recognized correclty, and one to apply the right attribute extravaganza.

Comment (3lvis) [inline Tests/MessageTests.swift]: @marijnschilling A separate test makes sense :)

**Comment by 3lvis** [comment]
BOOM!

**Comment by marijnschilling** [comment]
👏

**Pull request merged**